text/0000-buildpack-docs.md

```diff
@@ -0,0 +1,66 @@
+# Meta
+[meta]: #meta
+- Name: Buildpack.toml documentation
+- Start Date: (fill in today's date: 2021-04-26)
+- Author(s): (dwillist)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: "N/A"
+
+# Summary
+[summary]: #summary
+We propose adding a `documentation` file `README.md` adjacent to the `buildpack.toml` file in buildpacks.
+
+We would also like a new OCI label `io.buildpacks.buildpack.docs` in order to attach documentation on the outside of builder images and buildpackages.
+
+
+# Motivation
+[motivation]: #motivation
+
+- Why should we do this?
+  - To keep documentation accessible from the pack CLI & other buildpack tooling.
```

Comment:
```suggestion
  - To keep documentation accessible from the pack CLI and other buildpack tooling.
```
As far as I saw in other RFCs and documentations in general, we usually use `and` and not `&`.
If you agree, there are a few other places in this file that should be changed.